main.tf

```diff
@@ -1,5 +1,10 @@
 data "aws_region" "default" {}
 
+resource "random_string" "suffix" {
```

Comment:
Where is this used

Reply:
Oh i see it under takeover suffix. Can we rename this so it's clear it's only for takeover?

```suggestion
resource "random_string" "takeover_suffix" {
```